docs/integrations/fastapi.md

```diff
@@ -102,6 +102,44 @@ To avoid tracing certain URLs, you can specify a string of comma-separated regex
 - The environment variable `OTEL_PYTHON_FASTAPI_EXCLUDED_URLS`.
 - The environment variable `OTEL_PYTHON_EXCLUDED_URLS` (which will also apply to other instrumentation).
 
+!!! example
+    If you'd like to trace all URLs except the base `/` URL, you can use the following regex for `excluded_urls`: `^https?://[^/]+/$`
+
+    Breaking it down:
+
+    * `^` matches the start of the string
+    * `https?` matches `http` or `https`
+    * `://` matches `://`
+    * `[^/]+` matches one or more characters that are not `/` (this will be the host part of the URL)
+    * `/` matches `/`
+    * `$` matches the end of the string
+    
+    So this regex will only match routes that have no path after the host.
+
+    This instrumentation might look like:
+
+    ```py
+    from fastapi import FastAPI
+
+    import logfire
+
+    app = FastAPI()
+
+    logfire.configure()
+    logfire.instrument_fastapi(app, excluded_urls='^https?://[^/]+/$')
+
+    if __name__ == '__main__':
+        import uvicorn
+
+        uvicorn.run(app)
+    ```
+
+    If you visit http://127.0.0.1:8000/, that matches the above regex, so no span will be sent to logfire.
+    If you visit http://127.0.0.1:8000/hello/ (or any other endpoing that's not `/`, for that matter), a span will be sent to logfire.
```

Comment:
```suggestion
    If you visit http://127.0.0.1:8000/, that matches the above regex, so no tracing data will be recorded or sent to Logfire.
    If you visit http://127.0.0.1:8000/hello/ (or any other endpoing that's not `/`, for that matter), a trace will be started and sent to Logfire.
```
@alexmojaki is this right though? In particular, what happens if a user manually starts spans inside their endpoint, and isn't relying on the middleware. Does that misbehave because it just opens a trace below the middleware? I think what a user would _want_ to happen is that instrumentation is suppressed inside an excluded URL but I'm not sure if that's how it works now, and if it isn't, we should probably explain that..

Reply:
Yes, we should explain that it currently only prevents producing spans that come directly from the instrumentation.

Reply:
@alexmojaki, so do you think we should 1) apply the above suggestion and 2) add further clarification?

Reply:
I think @dmontagu's suggestion was meant to imply that it suppresses all tracing within, and we want to clarify that it doesn't, so I think no on the suggestion.

Reply:
Gotcha. I've reverted the suggestion and added an additional note.